package-lock.json

```diff
@@ -8549,7 +8549,7 @@
         "chromedriver": "bin/chromedriver"
```

Comment:
Looks like the chromedriver update to 118 has some dependencies on Node Version , Not sure if this will be issue but the Test are successful